charts/super-agent/README.md

```diff
@@ -0,0 +1,81 @@
+[![Community Plus header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Plus.png)](https://opensource.newrelic.com/oss-category/#community-plus)
+
+# super-agent
+
+![Version: 0.0.0-beta](https://img.shields.io/badge/Version-0.0.0--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+
+Bootstraps New Relic' Super Agent
+
+# Helm installation
+
+You can install this chart using directly this Helm repository:
+
+```shell
+helm repo add newrelic https://newrelic.github.io/helm-charts
+helm upgrade --install newrelic/CHART-TEMPLATE -f your-custom-values.yaml
```

Comment:
newrelic/CHART-TEMPLATE?

Reply:
This part is removed following the review of the chat downstream. Removed on [#d2e49c1](https://github.com/newrelic/helm-charts/pull/1175/commits/d2e49c17d289c548346804a27429633fbe279b38)